frontend/dispatcher.lua

```diff
@@ -682,6 +687,13 @@ function Dispatcher:getNameFromItem(item, settings, dont_show_value)
     return title
 end
 
+function Dispatcher:getActionArgsFunc(item)
```

Comment:
Dispatcher is not intended to provide such services.
Modules know their `action_name`, no need to ask the Dispatcher.